src/opencsg-1-fixes.patch

```diff
@@ -28,19 +28,20 @@ diff -ur OpenCSG-1.4.0.orig/example/example.pro OpenCSG-1.4.0/example/example.pr
  }
  
  HEADERS = displaylistPrimitive.h
-diff -ur OpenCSG-1.4.0.orig/src/src.pro OpenCSG-1.4.0/src/src.pro
---- OpenCSG-1.4.0.orig/src/src.pro	2014-09-15 22:25:33.000000000 +0200
-+++ OpenCSG-1.4.0/src/src.pro	2015-03-22 14:05:18.275029066 +0100
-@@ -2,10 +2,10 @@
- TARGET = opencsg
- VERSION = 1.4.0
+diff -ur OpenCSG-1.4.1.orig/src/src.pro OpenCSG-1.4.1/src/src.pro
+--- OpenCSG-1.4.1.orig/src/src.pro	2016-09-08 23:08:33.000000000 +0200
++++ OpenCSG-1.4.1/src/src.pro	2016-09-11 14:25:15.833496135 +0200
+@@ -6,10 +6,11 @@
+   INSTALLDIR = /usr/local
+ }
  
 -CONFIG += opengl warn_on release
 +CONFIG += opengl warn_on release link_pkgconfig
- INCLUDEPATH += ../include ../
+ INCLUDEPATH += ../include ../ $$INSTALLDIR/include
  CONFIG -= qt
--LIBS += -lGLEW
+-LIBS += -L$$INSTALLDIR/lib -lGLEW
++LIBS += -L$$INSTALLDIR/lib
 +PKGCONFIG += glew glut
```

Comment:
This comes from the original `Makefile`, I'm not sure for which scenario it's needed. I just tried to make minimal changes to that file, but if it makes things cleaner I can just remove that.

I've asked the author if he could include that patch upstream and he said he'll consider that, so maybe the patch will not be needed in later versions.